package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
What's the purpose of this new dependency ?

Reply:
To fix this warning
![npm-warning](https://github.com/user-attachments/assets/d4de8ddf-5923-4e2c-8d6e-4f65097d5622)